pro/src/components/pages/Offerers/Offerer/VenueV1/fields/PricingPoint/PricingPoint.tsx

```diff
@@ -135,7 +135,9 @@ const PricingPoint = ({
                       key={`venue-type-${venue.siret}`}
                       value={venue.nonHumanizedId}
                     >
-                      {`${venue.name} - ${venue?.siret}`}
+                      {venue.publicName
+                        ? `${venue.publicName} - ${venue?.siret}`
+                        : `${venue.name} - ${venue?.siret}`}
```

Comment:
(style) Je fais peu de JavaScript (et c'est un euphémisme !), donc je dis peut-être une bêtise : ne pourrait-on pas écrire plus simplement : 

```suggestion
                      {`${venue.publicName || venue.name} - ${venue?.siret}`}
```

(question) Au passage, je me demande à quoi sert le "?" dans `venue?.siret`. `venue` et `venue.siret` sont forcément définis ici, non ?